### PR TITLE
Create form customer and make validation feature

### PR DESCRIPTION
**Pull request by datdragon777**
## Description:
- Create form and make validation feature

## Storybook and app
image
image

### CODE REVIEW

File: practice-two/src/components/FormValidation/FormValidation.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    description: '',
+    address: '',
+  });
+
+  // Set value for form data
+  const handleChange = useCallback(
+    (e) => {
+      setFormData((prevFormData) => ({
+        ...prevFormData,
+        [e.target.name]: e.target.value,
```
Comment (lintran-agilityio) [inline practice-two/src/components/FormValidation/FormValidation.jsx:34]: cache the target event

File: practice-two/src/pages/Analytics/Analytics.jsx [hunk 2]
```diff
@@ -29,6 +29,12 @@ const Analytics = () => {
...
   const [isShowContextMenu, setIsShowContextMenu] = useState(false);
+  const [isShowForm, setIsShowForm] = useState(false);
+
+  // Handle open/close form
+  const handleShowForm = useCallback(() => {
+    setIsShowForm(!isShowForm);
+  }, [isShowForm]);
 
   // Event handler for clicking a customer
   const handleShowProfileInfo = (customer) => {
```
Comment (lintran-agilityio) [inline practice-two/src/pages/Analytics/Analytics.jsx:40]: check re-render for function

File: practice-two/src/components/FormValidation/FormValidation.jsx (hunk 1 above)
Comment (datdragon777) [reply practice-two/src/components/FormValidation/FormValidation.jsx:34]: image

File: practice-two/src/pages/Analytics/Analytics.jsx (hunk 2 above)
Comment (datdragon777) [reply practice-two/src/pages/Analytics/Analytics.jsx]: image

File: practice-two/src/components/Toast/Toast.jsx [hunk 3]
```diff
@@ -40,7 +40,7 @@ Toast.propTypes = {
 };
 
 Toast.defaultProps = {
-  message: MESSAGES.SUCCESSES.ADD_SUCCESSED,
+  message: MESSAGES.GET.SUCCESSES.ADD_SUCCESSED,
 };
 
 export default Toast;
```
Comment (lintran-agilityio) [inline practice-two/src/components/Toast/Toast.jsx:46]: check re-render Toast message

Comment (datdragon777) [reply practice-two/src/components/Toast/Toast.jsx:46]: image
I used memo for Toast component for avoiding re-render

**Pull request merged**